src/store.ts

```diff
@@ -227,28 +237,14 @@ export default function Store(
             this.SDKConfig.isDevelopmentMode = false;
         }
 
-        if (config.hasOwnProperty('v1SecureServiceUrl')) {
-            this.SDKConfig.v1SecureServiceUrl = config.v1SecureServiceUrl;
-        }
-
-        if (config.hasOwnProperty('v2SecureServiceUrl')) {
-            this.SDKConfig.v2SecureServiceUrl = config.v2SecureServiceUrl;
-        }
-
-        if (config.hasOwnProperty('v3SecureServiceUrl')) {
-            this.SDKConfig.v3SecureServiceUrl = config.v3SecureServiceUrl;
-        }
-
-        if (config.hasOwnProperty('identityUrl')) {
-            this.SDKConfig.identityUrl = config.identityUrl;
-        }
-
-        if (config.hasOwnProperty('aliasUrl')) {
-            this.SDKConfig.aliasUrl = config.aliasUrl;
-        }
-
-        if (config.hasOwnProperty('configUrl')) {
-            this.SDKConfig.configUrl = config.configUrl;
+        const endpoints: Dictionary = processEndpoints(
```

Comment:
I think this is more clear since `Dictionary` defaults to `any`.

```suggestion
        const endpoints: Dictionary<string> = processEndpoints(
```